# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/GfxInfo.h
# Commit: 6d7485c81e90
# Full Hash: 6d7485c81e90e2d2e70dffc5014bcaaf34b18a87
# Author: Robert Mader <robert.mader@posteo.de>
# Date: 2022-06-21 15:52:43
# Description:
#   Bug 1758473 - Move VA-API test into glxtest, r=stransky
#   
#   Some VA-API drivers are so broken that trying to use them
#   crashes Firefox. This is nothing entirely new for GPU drivers
#   and which is why we have `glxtest`.
# ==============================================================================

diff -r 54aff5bcc909 -r 6d7485c81e90 widget/gtk/GfxInfo.h
--- a/widget/gtk/GfxInfo.h	Tue Jun 21 08:19:37 2022 +0000
+++ b/widget/gtk/GfxInfo.h	Tue Jun 21 08:45:27 2022 +0000
@@ -111,6 +111,7 @@
   bool mIsXWayland;
   bool mHasMultipleGPUs;
   bool mGlxTestError;
+  bool mIsVAAPISupported;
 
   void AddCrashReportAnnotations();
 };